js/index.js

```diff
@@ -0,0 +1,102 @@
+function shuffle(array) {
+    let currentIndex = array.length,  randomIndex;
+
+    while (currentIndex !== 0) {
+
+        randomIndex = Math.floor(Math.random() * currentIndex);
+        currentIndex--;
+
+        [array[currentIndex], array[randomIndex]] = [
+            array[randomIndex], array[currentIndex]];
+    }
+
+    return array;
+}
+
+let expectedWord = []
+fetch('words.json')
+    .then(response => response.json())
+    .then(words => {
+        const wordsArr = words['fiveLetterWords']
+        const shuffledWords = shuffle(wordsArr)
+        expectedWord = shuffledWords[0].toUpperCase().split("")
+        console.log(expectedWord)
+        let attemptedWord = []
+        let characterCounter = 0
+// let attemptCounter = 0 - useful later
+
+// MAKE ON SCREEN LETTERS WORK
+        const onScreenLetters = document.querySelectorAll('.key')
+        const backspace = document.querySelector('.backspace')
+        const enter = document.querySelector('.enter')
+        onScreenLetters.forEach(function (letter) {
+            letter.addEventListener('click', function () {
+                if (attemptedWord.length < 5) {
+                    attemptedWord.push(letter.innerHTML)
+                    document.querySelector('.tile' + characterCounter).innerHTML = letter.innerHTML
+                    characterCounter++
+                    console.log(attemptedWord, characterCounter)
+                }
+            })
+        })
+
+        backspace.addEventListener('click', deleteLetter)
+        enter.addEventListener('click', enterPressed)
+
+        function deleteLetter (e) {
+            if (characterCounter > 0) {
+                attemptedWord = attemptedWord.slice(0, attemptedWord.length - 1)
+                characterCounter--
+                document.querySelector('.tile' + characterCounter).innerHTML = ''
+                // characterCounter--
+                console.log(attemptedWord, characterCounter)
+            }
+        }
+
+        function enterPressed () {
+            if (attemptedWord.length === 5) {
+                const result = checkWord(expectedWord,attemptedWord)
+                alert(result)
+            } else {
+                console.log('enter 5 characters')
+            }
+        }
```

Comment:
Please help remove the `else` part; `console.log` runs fine without it. 

Further, do you think we should consider removing the `console.log`? Nothing needs to happen if users presses enter before they finish making a guess.

Reply:
Amended!